llpc/test/shaderdb/core/OpExecutionModeId_TestLocalSizeId.spvasm

```diff
@@ -0,0 +1,99 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} final ELF info
+; SHADERTEST: COMPUTE_NUM_THREAD_X 0x0000000000000002
+; SHADERTEST: COMPUTE_NUM_THREAD_Y 0x0000000000000008
+; SHADERTEST: COMPUTE_NUM_THREAD_Z 0x0000000000000008
+; SHADERTEST: AMDLLPC SUCCESS
+; END_SHADERTEST
+
+
+; SPIR-V
+; Version: 1.4
+; Generator: Khronos Glslang Reference Front End; 10
+; Bound: 57
+; Schema: 0
+OpCapability Shader
+OpCapability GroupNonUniform
+%1 = OpExtInstImport "GLSL.std.450"
+OpMemoryModel Logical GLSL450
+OpEntryPoint GLCompute %4 "main" %14 %29 %43 %50
+OpExecutionModeId %4 LocalSizeId %7 %8 %9
+OpDecorate %7 SpecId 0
+OpDecorate %8 SpecId 1
+OpDecorate %9 SpecId 2
+OpDecorate %14 BuiltIn NumWorkgroups
+OpDecorate %16 SpecId 0
+OpDecorate %17 SpecId 1
+OpDecorate %18 SpecId 2
+OpDecorate %29 BuiltIn GlobalInvocationId
+OpDecorate %43 RelaxedPrecision
+OpDecorate %43 BuiltIn SubgroupSize
+OpDecorate %44 RelaxedPrecision
+OpDecorate %47 ArrayStride 4
+OpMemberDecorate %48 0 Offset 0
+OpDecorate %48 Block
+OpDecorate %50 DescriptorSet 0
+OpDecorate %50 Binding 0
+%2 = OpTypeVoid
+%3 = OpTypeFunction %2
+%6 = OpTypeInt 32 0
+%7 = OpSpecConstant %6 2
+%8 = OpSpecConstant %6 8
+%9 = OpSpecConstant %6 8
```

Comment:
Could you make the values for %8 and %9 different to make it easier to know that the value in the pal metadata is being taken from the correct place?

Reply:
Updated it. Thanks.